hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -134,7 +136,8 @@ public Response getClusterState() {
       totalKeys += fileRecord.getValue();
     }
     builder.setKeys(totalKeys);
-
+    // Subtract deleted containers from total containers.
+    containers = containers - deletedContainersCount;
```

Comment:
I think we should also expose in UI this deletedContainer count separately a part from maths computation and updating total container count.

Reply:
Got it I have exposed the `deletedContainersCount` to the response as well !!